comp/metadata/inventoryagent/README.md

```diff
@@ -6,7 +6,7 @@ This package populates some of the agent-related fields in the `inventories` pro
 This is enabled by default but can be turned off using `inventories_enabled` config.
 
 The payload is sent every 10min (see `inventories_max_interval` in the config) or whenever it's updated with at most 1
-update every 5 minutes (see `inventories_min_interval`).
+update every minute (see `inventories_min_interval`).
```

Comment:
The interval is still 5 minutes I think: https://github.com/DataDog/datadog-agent/blob/main/comp/metadata/internal/util/inventory_payload.go#L71